docs/concepts/qlight-node.md

```diff
@@ -0,0 +1,51 @@
+# GoQuorum qlight node
+
+A standard GoQuorum node will process all blocks and associated transactions; It also requires a local Private Transaction Manager to handle private data.
+This can require significant resources and may make it more difficult to scale networks.
+Additionally, since the node processes all transactions, privacy concerns could be raised by business partners.
+
+A Quorum qlight node can be deployed to reduce the amount of data that is available and that is shared by the full nodes with external parties.
+
+You can also use a qlight node for anything that could thrash the API, such as monitoring, debugging, state querying etc. This can alleviate any concerns around impact to the performance of a main node, or having to deal with network throttling due to third party network limits, or multiple clients all hitting the same main node.
+
+Qlight nodes have specific differences from standard quorum nodes, they:
+
+- Depend on a multi-tenancy (MT) 'server' node for receiving data and will only connect to the server node. There is no communication with any other node.
```

Comment:
Maybe you can add MPS here (in case security is not a concern)

Reply:
Done